linkerd/proxy/tap/src/accept.rs

```diff
@@ -75,8 +73,8 @@ impl Service<Connection<TcpStream>> for AcceptPermittedClients {
         Poll::Ready(Ok(()))
     }
 
-    fn call(&mut self, (meta, io): Connection<TcpStream>) -> Self::Future {
-        future::ok(match meta.peer_identity {
+    fn call(&mut self, ((peer_id, _), io): Connection<T, TcpStream>) -> Self::Future {
```

Comment:
this is a lot of tuples to pass around...but, introducing named structs for this might just make it messier, idk...

Reply:
i want to make this all more principled. this is an intermediate step...